clusters/app.ci/prow/03_deployment/retester.yaml

```
@@ -1,3 +1,23 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  labels:
+    app: retester
```

This is not mandatory. But let keep the consistency.

```suggestion
    app: prow
    component: retester
```